### DISCUSSION TITLE
一个检查 64 位系统的方法

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#3659] [讨论]
按住 Win+R 输入 msinfo32 
系统类型这里如果不是 ``` 基于 x64 的电脑 ``` 就是 32 位系统，和下面一致就是 64 位
![image](https://github.com/Hex-Dragon/PCL2/assets/160083863/2ff5506b-ea2d-45af-af56-89830356f4b5)
[龙猫说不打算支持 32 位了，以后版本可能是 64 位的了](https://github.com/Hex-Dragon/PCL2/issues/3649#issuecomment-2044656642)

**Comment by Ad-closeNN** [comment]
啥原因我cpu看的是x64而我只能用32位 _（奇怪的是我能用win7 x64而不能用win8.1+ x64）_
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/e16a3348-16e9-4681-ad74-3c0101487a3f)

**Reply by wuliaodexiaoluo** [reply]
不能吧，你要不去官网下个 x64 镜像重新安装系统？
正好把你无法解锁的情况覆盖的（每次安装都会生成新的识别码，说不定就随机到一个不会重复的？

**Comment by Pigeon0v0** [comment]
右键计算机 / 此电脑点属性
